packages/lib/surveyLogic/utils.ts

```
@@ -460,6 +460,18 @@ const evaluateSingleCondition = (
+      case "is":
+        return leftValue === rightValue;
+      case "isNot":
+        return leftValue !== rightValue;
+      case "isEmpty":
+        return leftValue === "";
+      case "isNotEmpty":
+        return leftValue !== "" && leftValue !== null && leftValue !== undefined;
+      case "isAnyOf":
+        if (Array.isArray(rightValue) && typeof leftValue === "string") {
+          return rightValue.includes(leftValue);
+        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)